go-controller/pkg/clustermanager/status_manager/status_manager_test.go

```diff
@@ -19,6 +21,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/utils/pointer"
```

Comment:
use `ptr` instead this one is deprecated. See https://github.com/ovn-org/ovn-kubernetes/pull/4207/commits/9fafe20d9a757c167904444c819e202f8a9560cd

Reply:
This is not yet deprecated with  `k8s.io/utils` version used in the PR, Moreover `pointer` is being referred in many other modules. so i think this would go in a separate PR.